src/gui/app.cpp

```diff
@@ -13,6 +13,65 @@
 
 namespace NeovimQt {
 
+struct ConnectorInitArgs {
+	enum Type {
```

Comment:
`enum class Type : int`

With enum class the compiler with enforce stronger type checks.